.github/workflows/build_rock.yaml

```diff
@@ -0,0 +1,169 @@
+name: Build Rocks
+
+on:
+  workflow_call:
+    inputs:
+      image-build-args:
+        description: |
+          List of build args to pass to the build image job
+        type: string
+        default: ""
+      registry:
+        description: Container Registrying top-level domain
+        type: string
+        default: ghcr.io
+      trivy-image-config:
+        type: string
+        description: Trivy YAML configuration for image testing that is checked in as part of the repo
+      working-directory:
+        type: string
+        description: The working directory for jobs
+        default: "./"
+        
+jobs:
+  get-runner-image:
+    name: Get runner image
+    uses: canonical/operator-workflows/.github/workflows/get_runner_image.yaml@main
+    with:
+      working-directory: ${{ inputs.working-directory }}
+  build-rocks:
+    name: Build rocks
+    uses: addyess/operator-workflows/.github/workflows/build_rocks.yaml@KU-52-reuse-gh-actions-from-canonical-to-build-amd-64-rocks-and-push-to-ghcr
```

Comment:
Cannot merge until this is first merged
https://github.com/canonical/operator-workflows/pull/243